code/controllers/subsystem/media.dm

```diff
@@ -28,15 +30,28 @@ SUBSYSTEM_DEF(media)
 		"mp3" = TRUE, //MPeg Layer 3 Container (And usually, Codec.)
 	)
 
+#define MEDIA_LOAD_FAILED -1
```

Comment:
why not just use 0

Reply:
Crash catching. `-1` means we at least returned under control, `null/0` means we returned without control, as in the json file is so bad it killed the proc.